src/components/Background.tsx

```diff
@@ -93,7 +93,7 @@ export default function Background({ children }: Props) {
         clearInterval(cometInterval)
       }
     }
-  }, [width, height])
+  }, [width])
```

Comment:
I've chosen to ignore the height.

This means that, when the users from desktop will increase the window height of their browsers, the background won't redraw itself.

This is a relatively little problem. This is something that only desktop users will see, and only when they'll resize the window only by editing the height, which doesn't happen very often.